skcosmo/selection/orthogonalizers.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+"""
+
+This module contains the necessary orthogonalizers for the CUR decomposition
+subselection method.
+
+"""
+
+# Authors: Rose K. Cersonsky
```

Comment:
I would put the authors in the module docstring instead, so that it is rendered in the documentation. I don't think there is a need to specify the license, since it is the same one as the full package.

Reply:
This one is still a comment